test/integration/install_test.go

```diff
@@ -670,6 +672,77 @@ func TestRetrieveUidPostUpgrade(t *testing.T) {
 	}
 }
 
+func TestOverridesSecret(t *testing.T) {
+
+	// Skip if Helm integration test
+	if TestHelper.IsHelm() {
+		t.Skip("Skipping as this is not a helm upgrade test")
+	}
```

Comment:
If you're introducing `TestHelper.IsHelm()`, would you update the other checks above to also us this same function?
Also, I think the code comment is not necessary :wink:

Reply:
I don't think we can replace the checks above. As in those cases, They want to skip only if the test is a helm upgrade test. In this case, We have to skip in all cases where Helm is involved as there is no overrides secret available with helm.

Also, updated the skip description to be more verbose.

Reply:
Sorry for my vague suggestion. I mean replacing `TestHelper.GetHelmReleaseName() (=|!)= ""` with `TestHelper.IsHelm()`. And while you're at it actually remove the `GetHelmReleaseName()` method declaration and the associated `helmReleaseName` property in `test_helper.go` as they won't be used at all,

Reply:
I removed `GetHelmReleaseName()` but was not able to remove `helm.releaseName` as the release name is used with helm upgrades, etc. So, Makes sense to have it still. WDYT?

Reply:
Okay, I changed it to use `GetHelmReleaseName` isntead of my new `IsHelm` as it was breaking some stuff. Looks like `releaseName` is a better toggle to rely on then helm binary